stock_picking_deliver_uos/test/stock_partial_picking.yml

```diff
@@ -3,13 +3,34 @@
 -
   !record {model: stock.picking.out, id: partial_picking_out}:
     partner_id: base.res_partner_2
-    move_lines:
-      - product_id: product.product_product_24
-        product_qty: 4
-        product_uos_qty: 5
-      - product_id: product.product_product_20
-        product_qty: 4
-        product_uos_qty: 5
+-
+  I add lines
+-
+  !python {model: stock.picking}: |
```

Comment:
you can pass the `id` like you did in the `!record` block above, and you get `self` as a new-api recordset. Normally a good idea.